.github/workflows/build.yml

```diff
@@ -3,33 +3,90 @@ name: Build
 on: push
 
 jobs:
-  pytest:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+        - "3.6"
+        - "3.7"
+        - "3.8"
+        - "3.9"
+        - "3.10"
```

Comment:
I would recommend not expanding the support matrix to older versions.

Reply:
I'm of the mind that it should test any version that the `toml` file indicates, otherwise we're ultimately guessing that it's still compatible.